### PR TITLE
doc: add the labels of mode bugtype

### PR DESCRIPTION
**Pull request by monperrus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ https://hacks.mozilla.org/2019/04/teaching-machines-to-triage-firefox-bugs/
 
 - **backout** - The aim of this classifier is to detect patches that might be more likely to be backed-out (because of build or test failures). It could be used for test prioritization/scheduling purposes.
 
-- **bugtype** - The aim of this classifier is to classify bugs according to their type.
+- **bugtype** - The aim of this classifier is to classify bugs according to their type, with four labels: crash/memory/performance/security.
```
Comment (marco-c) [inline README.md:15]: Those are the labels we could gather automatically from a subset of bugs on Bugzilla. The plan is to add more types after manual labeling.

Comment (monperrus) [reply README.md]: thanks. clarified the explanation.

File: README.md [hunk 2]
```diff
@@ -12,7 +12,8 @@ https://hacks.mozilla.org/2019/04/teaching-machines-to-triage-firefox-bugs/
 
 - **backout** - The aim of this classifier is to detect patches that might be more likely to be backed-out (because of build or test failures). It could be used for test prioritization/scheduling purposes.
 
-- **bugtype** - The aim of this classifier is to classify bugs according to their type.
+- **bugtype** - The aim of this classifier is to classify bugs according to their type. The  labels are gathered automatically from bugs: right now they are "crash/memory/performance/security". The plan is to add more types after manual labeling.
```
Comment (marco-c) [inline README.md:15]:
```suggestion
- **bugtype** - The aim of this classifier is to classify bugs according to their type. The labels are gathered automatically from bugs: right now they are "crash/memory/performance/security". The plan is to add more types after manual labeling.
```

File: README.md [hunk 3]
```diff
@@ -12,7 +12,8 @@ https://hacks.mozilla.org/2019/04/teaching-machines-to-triage-firefox-bugs/
 
 - **backout** - The aim of this classifier is to detect patches that might be more likely to be backed-out (because of build or test failures). It could be used for test prioritization/scheduling purposes.
 
-- **bugtype** - The aim of this classifier is to classify bugs according to their type.
+- **bugtype** - The aim of this classifier is to classify bugs according to their type. The labels are gathered automatically from bugs: right now they are "crash/memory/performance/security". The plan is to add more types after manual labeling.
+
```
Comment (marco-c) [inline README.md:16]:
```suggestion
```

**Comment by monperrus** [comment]
Great, thanks!

**Pull request merged**